src/NUnitFramework/framework/Assert.cs

```diff
@@ -40,7 +39,7 @@ public abstract partial class Assert
         /// <param name="a"></param>
         /// <param name="b"></param>
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public static new bool Equals(object a, object b)
+        public new static bool Equals(object a, object b)
```

Comment:
They should absolutely be in the classic.
But, since they only throws they're there more to prevent anyone using them.  A kind of safeguarding since they override the built-in ones. So not sure if we should remove them from the constraint Assert.

Reply:
About orderning:   MS raises IDE0036 wanting static before new, whereas resharper seems to follow google's ordering style (https://google.github.io/styleguide/csharp-style.html) which places new before static.  I have no strong opinion on this, so I guess we follow MS.